### DISCUSSION TITLE
Hello I have problem There are groups [Oct 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4247] [Discord Threads]
> 🙋‍♂️ **KVY**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### You can pass limit to get more records, for example if the LTAR field name is `Users` then you can pass query parameter like `nested[Users][limit]=50`  
#  
> 🙋‍♂️ **KVY**  
#### Is it right?  
![image0](https://cdn.discordapp.com/attachments/1031535105237336135/1031544028556169287/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/1031535105237336135/1031544028929466468/unknown.png)  
#  
> 🙋‍♂️ **KVY**  
#### I have empty response :((  
![image0](https://cdn.discordapp.com/attachments/1031535105237336135/1031544214707765278/unknown.png)  
#  
> 🙋‍♂️ **KVY**  
#### Oh its fields... sorry) Can I use nested[members][limit] query in such request ?  
![image0](https://cdn.discordapp.com/attachments/1031535105237336135/1031547309160398868/unknown.png)  
#  
> 👨‍💻 **pranavxc**  
#### Yes  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1031535105237336135)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
